typescript/taxonomy-api.ts

```diff
@@ -0,0 +1,905 @@
+/* tslint:disable */
+/* eslint-disable */
+// Generated using typescript-generator version 3.2.1263 on 2023-05-02 07:54:10.
+
+export interface Contexts {
+}
+
+export interface Context {
+    id: string;
+    path: string;
+    name: string;
+}
+
+export interface ContextPOST {
+    id: string;
+}
+
+export interface CrudController<T> {
+}
+
+export interface CrudControllerWithMetadata<T> extends CrudController<T> {
+}
+
+export interface NodeConnections extends CrudControllerWithMetadata<any> {
+    allNodeConnections: NodeConnection[];
+}
+
+export interface NodeConnectionPOST {
+    parentId: string;
+    /**
+     * Child id
+     */
+    childId: string;
+    /**
+     * Backwards compatibility: Always true. Ignored on insert/update
+     */
+    primary: boolean;
+    /**
+     * Order in which to sort the child for the parent
+     */
+    rank: number;
+    /**
+     * Relevance id
+     */
+    relevanceId: string;
+}
+
+export interface NodeConnectionPUT {
+    /**
+     * Connection id
+     */
+    id: string;
+    /**
+     * Backwards compatibility: Always true. Ignored on insert/update
+     */
+    primary: boolean;
+    /**
+     * Order in which subtopic is sorted for the topic
+     */
+    rank: number;
+    /**
+     * Relevance id
+     */
+    relevanceId: string;
+}
+
+export interface NodeResources extends CrudControllerWithMetadata<any> {
+    allNodeResources: NodeResource[];
+}
+
+export interface NodeResourcePOST {
+    /**
+     * Node id
+     */
+    nodeId: string;
+    /**
+     * Resource id
+     */
+    resourceId: string;
+    /**
+     * Primary connection
+     */
+    primary: boolean;
+    /**
+     * Order in which resource is sorted for the node
+     */
+    rank: number;
+    /**
+     * Relevance id
+     */
+    relevanceId: string;
+}
+
+export interface NodeResourcePUT {
+    /**
+     * Node resource connection id
+     */
+    id: string;
+    /**
+     * Primary connection
+     */
+    primary: boolean;
+    /**
+     * Order in which the resource will be sorted for this node.
+     */
+    rank: number;
+    /**
+     * Relevance id
+     */
+    relevanceId: string;
+}
+
+export interface NodeTranslations {
+}
```

Comment:
Skal denne være tom?

Reply:
Ja, for dette er controlleren som ligger bak `v1/node-translations`

Reply:
Fjerna controllere fra generert typescript.